tensorflow/python/keras/layers/core.py

```diff
@@ -145,6 +145,12 @@ class Dropout(Layer):
     inputs: Input tensor (of any rank).
     training: Python boolean indicating whether the layer should behave in
       training mode (adding dropout) or in inference mode (doing nothing).
+      
+  Usage Example:
+    >>> import tensorflow as tf
+    >>> model = tf.keras.models.Sequential()
+    >>> model.add(tf.keras.layers.Dropout(0.2, input_shape=(16,)))
+    >>> model.add(tf.keras.layers.Dense(1))
```

Comment:
Thanks for your contribution , can you please remove chevron(>>>) symbol . 
Here is the link for [contribution guidelines.](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#382-modules)

Reply:
Hi. I think I have removed them all.